blueprints/app/files/.eslintrc.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
     ecmaVersion: 2017,
     sourceType: 'module'
   },
-  extends: 'eslint:recommended',
+  extends: ['eslint:recommended', 'plugin:ember/recommended'],
+  plugins: ['ember'],
```

Comment:
can we move the `plugins` line above `extends`? feels strange to reference the plugin before it's mentioned

Reply:
Seems good to me, updated.